contributor-docs/release-guide.md

```diff
@@ -1010,14 +1010,14 @@ write to BigQuery, and create a cluster of machines for running containers (for
   * **Verify the hashes**
 
     ```
-    sha512sum -c apache-beam-2.5.0-python.tar.gz.sha512
-    sha512sum -c apache-beam-2.5.0-source-release.tar.gz.sha512
+    sha512sum -c apache_beam-2.5.0-python.tar.gz.sha512
```

Comment:
Take this change to update 2.5.0 -> 2.58.0 ? (2.5.0 had apache-beam name)

Reply:
actually, it's probably better to update this to apache_beam-2.XX.0 instead of using a specific version as an example. let me update that